CHANGELOG.rst

```diff
@@ -9,6 +9,13 @@ Added
 
 - Codecov configuration for coverage
 
+v8.1.1
+------
```

Comment:
nit: usually we bump as part of release.  fyi release currently rolls up _all_ changes landed on master

Reply:
Ah, my bad. Ok will revert.

Reply:
Should I create an "Unreleased"?
Like so: 
https://github.com/strateos/transcriptic/commit/29d5ae6c99e514d1883cfd5ae78266faa7e28a56#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43c

I'll do that....

Reply:
Ok, didn't see the "Unreleased" title.